hcmanager/utils.go

```diff
@@ -18,9 +18,10 @@ package hcmanager
 import (
 	"context"
 	"errors"
-	hwcc "hardware-classification-controller/api/v1alpha1"
 	"net"
 
+	hwcc "hardware-classification-controller/api/v1alpha1"
+
```

Comment:
this newline should not be here, and the block should then be in alphabetical order

Reply:
@maelk Done